src/visitors/global_var_visitor.cpp

```diff
@@ -0,0 +1,60 @@
+/*************************************************************************
+ * Copyright (C) 2018-2019 Blue Brain Project
+ *
+ * This file is part of NMODL distributed under the terms of the GNU
+ * Lesser General Public License. See top-level LICENSE file for details.
+ *************************************************************************/
+
+#include <iostream>
+#include <memory>
+
+#include "ast/global.hpp"
+#include "ast/program.hpp"
+#include "ast/range.hpp"
+#include "ast/statement_block.hpp"
+#include "visitors/global_var_visitor.hpp"
+/**
+ * \file
+ * \brief AST Visitor to change ast::GlobalVar to ast::Range
+ * variables in nmodl::symtab to nmodl::symtab::syminfo::NmodlType::range_var
+ * so that they can be handled by Codegen as RANGE variables
+ */
+
+namespace nmodl {
+namespace visitor {
+
+void GlobalToRangeVisitor::visit_statement_block(ast::StatementBlock& node) {
```

Comment:
typically there are lots of statement blocks and global variables are only defined in neuron block. So I am wondering why we need to visit all statement blocks? Visiting neuron block will suffice ?

Reply:
You're right. Visiting only the `NeuronBlock` should be enough. I will fix this